qiskit/transpiler/target.py

```diff
@@ -661,10 +732,20 @@ def operations(self):
     @property
     def instructions(self):
         """Get the list of tuples ``(:class:`~qiskit.circuit.Instruction`, (qargs))``
-        for the target"""
-        return [
-            (self._gate_name_map[op], qarg) for op in self._gate_map for qarg in self._gate_map[op]
-        ]
+        for the target
+
+        For globally defined variable width operations the tuple will be of the form
+        ``(class, None)`` where class is the actual operation class that
+        is globally defined.
+        """
+        res = []
+        for op in self._gate_map:
+            for qarg in self._gate_map[op]:
+                if not inspect.isclass(self._gate_name_map[op]):
+                    res.append((self._gate_name_map[op], qarg))
+                else:
+                    res.append((self._gate_name_map[op], None))
```

Comment:
Won't the `qarg` just automatically be `None` in the lookup here?  I'm not certain we need the special handling?

Reply:
You're right it will be, this is leftover from an earlier draft where I was thinking the properties payload for the target would be `None` instead of `{None: None}`. But I can revert this change and just use the list comprehension from before without issue.

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/8852/commits/17ee512e87676059faee21a21e62c6afbec35201